File: logica-de-programacao/secao-9/exercicios/retangulo.c
#include <stdio.h>
#include <math.h>


void limpar_entrada() {
    char c;
    while ((c = getchar()) != '\n' && c != EOF) {}
}


int main() {

    double b, a, area, perimetro, diagonal;

    printf("Base do retangulo: ");
    scanf("%lf", &b);

    printf("Altura do retangulo: ");
    limpar_entrada();
    scanf("%lf", &a);

    area = b * a; 

    perimetro = (b*2) + (a*2);

    diagonal = sqrt(pow(b, 2) + pow(a, 2)); 


    printf("AREA = %.4lf\n", area);
    printf("PERIMETRO = %.4lf\n", perimetro);
    printf("DIAGONAL = %.4lf\n", diagonal);

    return 0;
}

File: logica-de-programacao/secao-12/exercicios/retangulo.cs
using System.Globalization;

CultureInfo ci = CultureInfo.InvariantCulture;

double b, h, area, perimetro, diagonal;

Console.Write("Base do retangulo: ");
b = double.Parse(Console.ReadLine(), ci);

Console.Write("Altura do retangulo: ");
h = double.Parse(Console.ReadLine(), ci);

area = b * h;
perimetro = (2 * h) + (2 * b);
diagonal = Math.Sqrt(Math.Pow(b, 2) + Math.Pow(h, 2));

Console.WriteLine("AREA = " + area.ToString("F4", ci));
Console.WriteLine("PERIMETRO = " + perimetro.ToString("F4", ci));
Console.WriteLine("DIAGONAL = " + diagonal.ToString("F4", ci));